src/redisraft.c

```diff
@@ -670,8 +670,20 @@ static void handleRedisCommandAppend(RedisRaftCtx *rr,
         return;
     }
 
+    if (cmd_flags & CMD_SPEC_SEND_ACL) {
+        RedisModuleString *user_name = RedisModule_GetCurrentUserName(ctx);
+        RedisModuleUser *user = RedisModule_GetModuleUserFromUserName(user_name);
+        RedisModule_FreeString(ctx, user_name);
+        cmds->acl = RedisModule_GetModuleUserACLString(ctx, user);
+        RedisModule_FreeModuleUser(user);
```

Comment:
Probably, we should consider something like `RedisModule_GetCurrentUserAcl()` just to avoid these calls and allocations.

Reply:
ok.  change to redis, easy enough.